paper-input-container.html

```diff
@@ -239,22 +266,31 @@
       'input': '_onInput'
     },
 
+    get _valueChangedEvent() {
+      return this.attrForValue + '-changed';
+    },
+
+    get _propertyForValue() {
+      return Polymer.CaseMap.dashToCamelCase(this.attrForValue);
+    },
+
     attached: function() {
       var input = Polymer.dom(this).querySelector('input,.paper-input-input');
-      this._inputHasContent = !!input.value;
-      this._inputIsInvalid = !input.validity.valid;
+      this._handleInput(input);
+      // once more for add-ons, async to wait for distribution
```

Comment:
I think since the elements are made ready top-down, the child nodes aren't ready at this time to get the event in `_handleInput`.